### ISSUE TITLE
What is the output prediction means ?

### ISSUE THREAD

**Issue by mailong25** [Refefer/fastxml#15]
I tired to use python API:
X = [Sparse or numpy arrays]
y = [[1, 3]] # Currently requires list[list[int]]

trainer = Trainer(n_trees=32, n_jobs=-1)

trainer.fit(X, y)

trainer.save(path)

clf = Inferencer(path)

clf.predict(X, fmt='dict')

And the prediction result of an example look like this:
[(5866, -0.40310976),
 (437, -0.67100734),
 (995, -0.8778681),
 (2642, -1.1181042),
 (5217, -1.1278155),
 (5967, -1.1540765),
 (7558, -1.3282802),
 (4391, -1.5430373),
 (4017, -1.624005),
 (5781, -1.9639409),
 (1347, -2.012597),
 (4063, -2.0736518)

What -0.40310976, -0.67100734, ..... mean ? 
And how can I select the most appropriate labels from this result?

**Comment by Refefer** [comment]
That the log likelihood.  It's sorted in descending order of prediction, so
the first label is the best.
